Lib/test/test_enum.py

```diff
@@ -26,11 +25,11 @@
 
 def load_tests(loader, tests, ignore):
     tests.addTests(doctest.DocTestSuite(enum))
-    if os.path.exists('Doc/library/enum.rst'):
-        tests.addTests(doctest.DocFileSuite(
-                '../../Doc/library/enum.rst',
-                optionflags=doctest.ELLIPSIS|doctest.NORMALIZE_WHITESPACE,
-                ))
+    tests.addTests(doctest.DocFileSuite(
```

Comment:
the whole test library is shipped to the user, while the docs may not.

Keep the behavior as is please.

Reply:
I agree that docs may not be present, but I see that path that is passed to `os.path.exists` should be changed to "../../Doc/library/enum.rst" anyway. Not sure about `module_relative` flag though.

Reply:
Yeah, we need to figure out the correct path.